friends-in-the-nineteenth-century-volume-1/modernized/08-chapter-7.adoc

```diff
@@ -377,11 +403,14 @@ But the meeting at length decided that the Committee on the General
 Epistle should be directed to prepare and introduce a paragraph,
 fully and explicitly confirmatory of the well-known and
 previously recorded sentiments of the Society on the subject.`"^
-footnote:[Stephens`'s Report of London Yearly Meeting. 1836, page 18.]
+footnote:[Stephens`'s _Report of London Yearly Meeting._ 1836, page 18.]
 The committee, however, appear to have transcended the terms of their instructions,
 and at a subsequent sitting brought in a form for the paragraph,
 which after some slight changes stood as follows:^
-footnote:[Ibid., page 30.]
+footnote:[_Ibid.,_ page 30.]
+
+[.embedded-content-document]
```

Comment:
I would probably put the long quotation below as a block quote, since it seems to have been taken from a published report of the yearly meeting. My basic rule (although I sometimes break it) is that I use block quotes for chunks of published text, and the embedded sections for letters, epistles, addresses, testimonies, legal papers (and the other things on the list in the style guide).